### PR TITLE
MCSC-653 Unsupported affinity group error page

### PR DESCRIPTION
**Pull request by TJSarno**

### CODE REVIEW

**Review by MichaelAAbbott — DISMISSED** [review]
Amazing, great stuff! :+1:

File: conf/messages.en [hunk 1]
```diff
@@ -467,3 +467,9 @@ upliftFailed.evidence = If you cannot give us evidence to confirm your identity,
 upliftFailed.contact = You can contact us to access our services and speak to an advisor. Our advisors can help you finish your task and give you information over the phone or via web chat. They cannot complete online identity verification for you.
 upliftFailed.details = Alternatively, give us details from one or two of the evidence types listed, and try again to confirm your identity online.
 upliftFailed.support = We also offer technical support if your details do not match our records.
+
+cannotUseServiceNotIndividual.title = Cannot use submission service - Calculate your public service pension adjustment - GOV.UK
```
Comment (ElliotAndersonHMRC) [inline conf/messages.en:471]: "- Calculate your public service pension adjustment - GOV.UK" should be removed as this is automatically added to the end of the title

**Review by ElliotAndersonHMRC — CHANGES_REQUESTED** [review]
One small fix

**Review by ElliotAndersonHMRC — DISMISSED** [review]
Looks good!

File: app/views/UnauthorisedView.scala.html [hunk 2]
```diff
@@ -22,7 +22,8 @@
 
 @layout(
     pageTitle = titleNoForm(messages("unauthorised.title")),
-    timeout   = false
+    timeout   = false,
+    showSignOut = false
```
Comment (dacohm) [inline app/views/UnauthorisedView.scala.html:26]: We still want a sign out link since there is a need to sign out from GG.

**Review by dacohm — APPROVED** [review]

**Pull request merged**